data-generation/generators/generateUsers.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+const faker = require('faker');
+const fs = require('file-system');
+
+const writeUsers = fs.createWriteStream('../generatedData/users.csv');
+writeUsers.write('user_id,username,email,location\n', 'utf8');
+
+function writeTenMillionUsers(writer, encoding, callback) {
+  // let i = 5000000; // how many units do you want? End goal is 10 mill
+
+  function write() {
+    let ok = true;
+    do {
+      i -= 1;
+
+      const user_id = faker.random.uuid();
+      const preModifiedEmail = faker.internet.email();
+      const email = preModifiedEmail.toLowerCase();
+      const username = preModifiedEmail.split('@')[0];
+      const location = `${faker.address.city()}, ${faker.address.stateAbbr()}`;
```

Comment:
I see you used the fakers address functions. How long did it take you to run the user script?